src/pages/APIPage.tsx

```diff
@@ -71,7 +71,7 @@ export const APIPage: FunctionComponent = () => {
         <LanguageProvider>
         <Helmet>
             <title>{selectedApi.displayName} | {Config.title} </title>
-            <meta name="rhd:node-type" content="api_docs" />
+            <meta name="rhd:node-type" content="api_catalog" />
```

Comment:
This is an internal id, we should keep the `api_docs` unless developers.redhat.com team tell us otherwise.

```suggestion
            <meta name="rhd:node-type" content="api_docs" />
```

Reply:
Alistair Mcgranaghan left a comment on the JIRA ticket requesting this to be changed.